src/nanotron/config/models_config.py

```diff
@@ -50,6 +50,7 @@ class LlamaConfig:
     tie_word_embeddings: bool = False
     use_cache: bool = True
     vocab_size: int = 32000
+    for_inference: bool = False  # if true, replace TritonRMSNorm with LayerNorm for a fixed output. use TritonRMSNorm for training as it's faster
```

Comment:
i don't think we should have that in the model config. just in the `run_generate.py` file is fine

Reply:
I plan to add it as an optional argument in the LlamaModel and LlamaModel class __init__ function.

Reply:
done